pkg/analysis/optionalfields/testdata/src/b/a.go

```diff
@@ -171,6 +171,16 @@ type A struct {
 	// +optional
 	StructWithMinProperties B `json:"structWithMinProperties,omitempty"` // want "field StructWithMinProperties is optional and should be a pointer"
 
+	// structWithMinPropertiesAndOmitEmptyAndOmitZero is a struct field with a minimum number of properties and has both omitzero and omitemtpy tag.
+	// +kubebuilder:validation:MinProperties=1
+	// +optional
+	StructWithMinPropertiesAndOmitEmptyAndOmitZero B `json:"structWithMinPropertiesAndOmitEmptyAndOmitZero,omitempty,omitzero"`
+
+	// structWithOnlyOmitZeroTag is a struct field with a minimum number of properties and has only omitzero tag.
+	// +kubebuilder:validation:MinProperties=1
+	// +optional
+	StructWithOnlyOmitZeroTag B `json:"structWithOnlyOmitZeroTag,omitzero"`
```

Comment:
IMO, if we aren't using the ignore policy on `omitempty`, we should add `omitempty` as well as `omitzero` to be consistent with other types